cmd/plan.go

```diff
@@ -77,7 +77,13 @@ func planInstall(rv cran.RVersion, exitOnMissing bool) (*cran.PkgNexus, gpsr.Ins
 	startTime := time.Now()
 
 	//Check library existence
-	libraryExists, _ := afero.DirExists(fs, cfg.Library)
+	libraryExists, err := afero.DirExists(fs, cfg.Library)
+
+	if err != nil {
+		log.WithFields(log.Fields{
```

Comment:
panic(err)

Reply:
after discussion, we should instead use log.Error to still communicate the error loudly but not crash as to stop the plan in case downstream elements still need to use plan info